streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -662,6 +664,16 @@ object StreamingContext extends Logging {
     }
   }
 
+  /**
+   * Get the context that is currently active, that is, started but not stopped.
+   */
+  @Experimental
+  def getActive(): Option[StreamingContext] = {
+    ACTIVATION_LOCK.synchronized {
+      Option(activeContext.get())
```

Comment:
The locking structure is going to be fixed. I am waiting for #6060  to be merged before I can take a structured look.